### PR TITLE
Jj emantomo 2.99.52 pppt

### PR DESCRIPTION
**Pull request by JorMaister**

### CODE REVIEW

**Comment by azazellochg** [comment]
holy moly, good luck @pconesa xD

File: emantomo/__init__.py [hunk 1]
```diff
@@ -25,164 +23,13 @@
...
+import eman2
 
-import pwem
-import pyworkflow.utils as pwutils
-from scipion.install.funcs import VOID_TGZ
-
-import emantomo.constants as emanConst
-
-__version__ = "3.1.5"
+__version__ = "3.1.6"
```
Comment (pconesa) [inline emantomo/__init__.py:28]: Should be 3.2.0

Comment (JorMaister) [reply emantomo/__init__.py:28]: You're right, it's a Minor one, not a patch.

File: emantomo/__init__.py [hunk 2]
```diff
@@ -24,165 +22,15 @@
...
-__version__ = "3.1.5"
+__version__ = "3.2.0"
 _logo = "eman2_logo.png"
 _references = ['GALAZMONTOYA2015279', 'BELL201625']
 _url = "https://github.com/scipion-em/scipion-em-emantomo"
 
+try:
+    import eman2
+    class Plugin(eman2.Plugin):
+        pass
```
Comment (pconesa) [inline emantomo/__init__.py:33]: I'd implement here _defeineBinaries as an empty method to avoid double definition of binaries

**Review by pconesa — CHANGES_REQUESTED** [review]

**Review by pconesa — APPROVED** [review]

**Pull request merged**